src/main/java/com/qaprosoft/carina/demo/phpTravels/components/NavigationBar.java

```diff
@@ -0,0 +1,53 @@
+package com.qaprosoft.carina.demo.phpTravels.components;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+import com.qaprosoft.carina.demo.phpTravels.pages.BookingsPage;
+import com.qaprosoft.carina.demo.phpTravels.pages.DashboardPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class NavigationBar extends AbstractUIObject {
+
+    @FindBy(xpath = "//i[normalize-space()='menu']")
+    private ExtendedWebElement leftNavMenu;
```

Comment:
leftNavMenu is a component, it should be LeftMenuBar type, not ExtendedWebElement